spec/core.md

```diff
@@ -27,6 +27,7 @@ The ABNF definition of a `did:webs` DID is as follows:
 webs-did = "did:webs:" domain-name ":" aid
 webs-did = "did:webs:" domain-name * (":" path) ":" aid
 ```
+`ABNF` stands for Augmented Backus-Naur Form, which is a formal notation used for describing syntax rules in various kinds of languages and protocols. We need to ensure that our implementation aligns with W3C standards and the ABNF rules specified in the DID documentation. These rules serve as a kind of "grammar" for DIDs, ensuring that they are formatted and used consistently.
```

Comment:
If DID core refers to ABNF, then we should link there instead of adding this paragraph. Can you double check?

Reply:
Yes, that's what I intend to do: one sentence + link. Some terms are just too complicated to define in one sentence. But it's what I strive for. See my Termtext PR72.
Btw: Feel free to add / change stuff you don't like afterwards. It's getting messy now with too may PRs, please accept a few first / then modify according to your wishes and then I'll be back on the front ;)